packages/perseus/src/interactive2/__tests__/movable.test.ts

```diff
@@ -0,0 +1,182 @@
+/* eslint-disable testing-library/prefer-user-event */
+import {fireEvent} from "@testing-library/react";
+
+import {Graphie} from "../../util/graphie";
+import Movable from "../movable";
+
+// Yay for side-effect imports!
+// eslint-disable-next-line import/no-unassigned-import
+import "../../util/interactive";
+
+describe("Movable", () => {
+    it("should be constructable", () => {
+        const movable = new Movable(
+            new Graphie(document.createElement("div")),
+            {},
+        );
+
+        expect(movable).toBeDefined();
+    });
+
+    it("should have accessor properties", () => {
+        const movable = new Movable(
+            new Graphie(document.createElement("div")),
+            {},
+        );
+
+        expect(typeof movable.isHovering).toBe("function");
+        expect(typeof movable.cursor).toBe("function");
+        expect(typeof movable.added).toBe("function");
+        expect(typeof movable.isHovering).toBe("function");
+        expect(typeof movable.isMouseOver).toBe("function");
+        expect(typeof movable.isDragging).toBe("function");
+        expect(typeof movable.mouseTarget).toBe("function");
+    });
+
+    it("cloneState()", () => {
```

Comment:
Jest provides `test` as an alias for `it`. Using `test` might make some of these descriptions read a bit nicer.
```suggestion
    test("cloneState()", () => {
```

Reply:
True.  TBH, `it()` and `test()` dissolve into the background when I look at test files.  I'm open to changing this for others though.